cupy/cuda/cupy_cub.cu

```diff
@@ -10,6 +10,8 @@
 #include <cub/device/device_histogram.cuh>
 #include <cub/iterator/counting_input_iterator.cuh>
 #include <cub/iterator/transform_input_iterator.cuh>
+#include <cuda/functional>
+#include <cuda/std/functional>
```

Comment:
Q: Do we need both?

Reply:
Yes. `<cuda/functional>` for `cuda::maximum<>` etc. and `<cuda/std/functional>` for `cuda::std::multiplies<>`.